cmake/find_target_cuda_dependencies.cmake

```diff
@@ -0,0 +1,13 @@
+enable_language(CUDA)
+
+find_dependency(CUDAToolkit REQUIRED)
+
+#if( QUDA_NVSHMEM )
```

Comment:
I tested this with Chroma and NVSHMEM enabled and it builds and runs fine with these lines commented out.